python/cugraph/cugraph/tests/mg/test_mg_property_graph.py

```diff
@@ -959,6 +959,50 @@ def test_add_data_noncontiguous():
         )
 
 
+@pytest.mark.parametrize("props_to_fill", ["vertex", "edge", "both"])
+def test_fillna(dask_client, props_to_fill):
```

Comment:
I'm assuming this test needs to be updated too for `fillna_vertices`/`fillna_edges`?

Reply:
yes, this will get updated shortly

Reply:
done